conf/routes

```diff
@@ -4,112 +4,205 @@ GET        /index.html                 @org.sagebionetworks.bridge.play.controll
 GET        /mobile/verifyEmail.html    @org.sagebionetworks.bridge.play.controllers.ApplicationController.verifyEmail
 GET        /mobile/resetPassword.html  @org.sagebionetworks.bridge.play.controllers.ApplicationController.resetPassword
 
+# Authentication
+POST   /v3/auth/signIn                  @org.sagebionetworks.bridge.play.controllers.AuthenticationController.signInV3
+POST   /v3/auth/signOut                 @org.sagebionetworks.bridge.play.controllers.AuthenticationController.signOutV3
+POST   /v3/auth/requestResetPassword    @org.sagebionetworks.bridge.play.controllers.AuthenticationController.requestResetPasswordV3
+POST   /v3/auth/resetPassword           @org.sagebionetworks.bridge.play.controllers.AuthenticationController.resetPasswordV3
+POST   /v3/auth/signUp                  @org.sagebionetworks.bridge.play.controllers.AuthenticationController.signUpV3
+POST   /v3/auth/verifyEmail             @org.sagebionetworks.bridge.play.controllers.AuthenticationController.verifyEmailV3
+POST   /v3/auth/resendEmailVerification @org.sagebionetworks.bridge.play.controllers.AuthenticationController.resendEmailVerificationV3
+
+# Users
+POST   /v3/users                  @org.sagebionetworks.bridge.play.controllers.UserManagementController.createUserV3
+POST   /v3/users/email            @org.sagebionetworks.bridge.play.controllers.StudyController.sendStudyParticipantsRosterV3
```

Comment:
I feel like POST /v3/users/email doesn't really describe what this API does. I think we should have a name with "sendParticipantRoster" somewhere in there.